core/app/models/spree/tax_rate.rb

```diff
@@ -129,12 +125,31 @@ def default_zone_or_zone_match?(order_tax_zone)
 
     private
 
-    def create_label
-      label = ""
-      label << (name.present? ? name : tax_category.name) + " "
-      label << (show_rate_in_label? ? "#{amount * 100}%" : "")
-      label << " (#{Spree.t(:included_in_price)})" if included_in_price?
-      label
+    def adjustment_label(amount)
+      Spree.t translation_key(amount),
+              scope: "adjustment_labels.tax_rates",
+              name: name.presence || tax_category.name,
+              amount: amount_for_adjustment_label
+    end
+
+    def amount_for_adjustment_label
+      return "" unless show_rate_in_label?
+      " " + ActiveSupport::NumberHelper::NumberToPercentageConverter.convert(
```

Comment:
In that case I could possible also remove the `show_rate_in_label` column, as that's be handled by `I18n` entirely, no?